scripts/mock_sds_endpoint.py

```diff
@@ -0,0 +1,135 @@
+                        ],
+                    },
+                    "guidance_exclude": {
+                        "title": "Exclude",
+                        "list": [
+                            "sports holdalls, gloves, clothing of textile materials, footwear, protective eyewear, rackets, balls, skates",
+                            "for skiing, water sports, golf, fishing', for skiing, water sports, golf, fishing, table tennis, PE, gymnastics, athletics",
+                        ],
+                    },
+                    "value_sales": {
+                        "answer_code": "89929001",
+                        "label": "Value of sales",
+                    },
+                    "volume_sales": {
+                        "answer_code": "89929002",
+                        "label": "Volume of sales",
+                        "unit_label": "Tonnes",
+                    },
+                    "total_volume": {
+                        "answer_code": "89929005",
+                        "label": "Total volume produced",
+                        "unit_label": "Tonnes",
+                    },
+                },
+                {
+                    "identifier": "201630601",
+                    "name": "Other Minerals",
+                    "cn_codes": "5908 + 5910 + 591110 + 591120 + 591140",
+                    "guidance_include": {
+                        "title": "Include",
+                        "list": [
+                            "natural graphite",
+                            "quartz for industrial use",
+                            "diatomite; magnesia; feldspar",
+                            "magnesite; natural magnesium carbonate",
+                            "talc including steatite and chlorite",
+                            "unexpanded vermiculite and perlite",
+                        ],
+                    },
+                    "guidance_exclude": {
+                        "title": "Exclude",
+                        "list": ["natural quartz sands"],
+                    },
+                    "value_sales": {
+                        "answer_code": "201630601",
+                        "label": "Value of sales",
+                    },
+                    "volume_sales": {
+                        "answer_code": "201630602",
+                        "label": "Volume of sales",
+                        "unit_label": "Kilogram",
+                    },
+                    "total_volume": {
+                        "answer_code": "201630605",
+                        "label": "Total volume produced",
+                        "unit_label": "Kilogram",
+                    },
+                },
+            ]
+        },
+    },
+}
+
+
+@app.route("/v1/unit_data")
+def get_sds_data():
+    dataset_id = request.args.get("dataset_id")
+
+    if dataset_id == "002":
+        return SUPPLEMENTARY_DATA_PRODCOM_PAYLOAD
+    return SUPPLEMENTARY_DATA_TILES_AND_SLATE_PAYLOAD
```

Comment:
have an elif for the other payload so you can also test what happens if the data set id is not valid?

Reply:
Updated this 👍